packages/lodestar/src/chain/interface.ts

```diff
@@ -105,4 +112,8 @@ export interface IBeaconChain {
    * Pre-process and run the per slot state transition function
    */
   receiveBlock(signedBlock: SignedBeaconBlock, trusted?: boolean): Promise<void>;
```

Comment:
Can we change this to return `void` so it's not misleading? the behaviour is very different to `processChainSegment`

Reply:
I'd say `yes`, would like @wemeetagain to confirm through. I can to create a separate PR if we want to do that.

at least what I see is the name `receive*` vs. `process*` is the main difference between these 2 apis.